### PR TITLE
Improve GasPriceComponent

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-commons/lib/core/constants/gas.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import {utils} from 'ethers';
+
+export const INITIAL_GAS_PARAMETERS = {
+  gasToken: '0x00...00',
```
Comment (JustynaBroniszewska) [inline universal-login-commons/lib/core/constants/gas.ts]: use constant `ETHER_NATIVE_TOKEN`

File: universal-login-react/src/ui/commons/GasPrice.tsx [hunk 2]
```diff
@@ -4,28 +4,32 @@ import './../styles/gasPriceDefault.sass';
...
 
-export const GasPrice = ({sdk, setGasTokenAddress, setGasPrice}: GasPriceProps) => {
+export const GasPrice = ({sdk, onGasParametersChanged}: GasPriceProps) => {
   const [gasModes] = useAsync(() => sdk.getGasModes(), []);
   const [modeName, setModeName] = useState<string>('');
   const [tokenAddress, setTokenAddress] = useState<string>('');
 
   const onModeChanged = (name: string) => {
     setModeName(name);
-    setGasPrice(getGasPriceFor(gasModes!, name, tokenAddress));
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/GasPrice.tsx]: What if there is no `gasModes`? We should use `ensure(gasModes)`

Comment (xitronix) [reply universal-login-react/src/ui/commons/GasPrice.tsx]: If no gasMode returm Loading

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/commons/GasPrice.tsx]: My bad

File: universal-login-sdk/lib/core/services/GasModeService.ts [hunk 3]
```diff
@@ -23,8 +23,8 @@ export class GasModeService {
     const tokensPrices = await this.priceObserver.getCurrentPrices();
 
     return [
-      await this.getMode('cheap', gasPrices.cheap, tokensPrices),
-      await this.getMode('fast', gasPrices.fast, tokensPrices)
+      this.createMode('cheap', gasPrices.cheap, tokensPrices),
+      this.createMode('fast', gasPrices.fast, tokensPrices)
```
Comment (xitronix) [inline universal-login-sdk/lib/core/services/GasModeService.ts]: I am not sure if I need rename getModes => createModes too. I guess no.

Comment (marekkirejczyk) [reply universal-login-sdk/lib/core/services/GasModeService.ts]: createMode looks good to me

**Comment by marekkirejczyk** [comment]
Needs rebase @xitronix

**Pull request merged**